com.unity.render-pipelines.universal/Shaders/CameraMotionVectors.shader

```diff
@@ -54,15 +54,20 @@ Shader "Hidden/kMotion/CameraMotionVectors"
             // Fragment
             half4 frag(Varyings input, out float outDepth : SV_Depth) : SV_Target
             {
+                UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
                 // Calculate PositionInputs
                 half depth = LoadSceneDepth(input.position.xy).x;
                 outDepth = depth;
-                half2 screenSize = _ScreenSize.zw;
-                PositionInputs positionInputs = GetPositionInput(input.position.xy, screenSize, depth, UNITY_MATRIX_I_VP, UNITY_MATRIX_V);
 
-                // Calculate positions
-                float4 previousPositionVP = mul(_PrevViewProjM, float4(positionInputs.positionWS, 1.0));
-                float4 positionVP = mul(UNITY_MATRIX_VP, float4(positionInputs.positionWS, 1.0));
+                // Transform to world space
+                float2 ndcpos = input.position.xy *= _ScreenSize.zw;
```

Comment:
There seems to have some confusion about _ScreenSize for URP. https://unity.slack.com/archives/C01GUUQR2H4/p1620209716210500?thread_ts=1620204487.203900&cid=C01GUUQR2H4
 We should fix _ScreenSize, meanwhile you can use _ScaledScreenParams.xy( be careful _ScaledScreenParams.zw is not inverse of .xy!)